src/web/components/Onwards/Carousel/CarouselCard.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Card } from '@frontend/web/components/Card/Card';
+import { LI } from '@frontend/web/components/Card/components/LI';
+
+type CarouselCardProps = {
+	trail: TrailType;
+};
+
+export const CarouselCard: React.FC<CarouselCardProps> = ({
+	trail,
+}: CarouselCardProps) => {
+	const {
+		pillar,
+		design,
+		url: linkTo,
+		image: imageUrl,
+		headline: headlineText,
+		webPublicationDate,
+		kickerText,
+	} = trail;
```

Comment:
I feel like these should be individual props rather than all just in a `trail` prop

Reply:
Yeah, I think you're right. We have a bit of a (unwritten) rule that explicit props is better. Eg. we ban prop spreading.

It'd be nice to be able to _know_ what values are actually being used from `trail` and which are required without having to dive in